src/successModal/elements.tsx

```diff
@@ -23,10 +23,9 @@ const StyledSuccessModal = styled(Modal)`
 
   .kirk-modal-body {
     display: flex;
+    min-height: -webkit-fill-available;
```

Comment:
Removing this to make it consistent with other modal and using min-height: 100%.
Tested in Safari, Firefox, Chrome desktop. And Safari & Chrome mobile, I didn't see any issue even with the address bar, I'm guessing since we're in % it's working, it wouldn't if we were using vh.